src/models/DownloadLimitAction.js

```diff
@@ -57,8 +57,17 @@ export default class DownloadLimitAction {
 	get handlers() {
 		return {
 			'update:value': debounce(async (limit) => {
+				console.debug('[DEBUG]', appName, 'Setting limit of ' + this._store.token + ' to ' + limit)
 				this._store.loading = true
-				await setDownloadLimit(this._store.token, limit)
+
+				// If the value is not correct, let's remove the limit
+				if (!parseInt(limit) || parseInt(limit) < 1) {
```

Comment:
same here: when something that is not an int was entered, we should not delete the count, imo.
Only if the field is empty or less then 1, imo

Reply:
okay, I understand why you implemented it like this.